Assets/Scripts/PlayerCharacter/PlayerHPDecreasement.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Combat;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(CombatEntity))]
+public class PlayerHPDecreasement : MonoBehaviour
+{
+    [SerializeField] public int hpDecreasingValue = 1;
```

Comment:
You do not need this `SerializeField` attribute on public fiels that are serializable. Unity serializes those fields automatically, and that is why you can see the values in the inspector.